config/superblocks.ts

```diff
@@ -68,9 +68,10 @@ export const superBlockOrder: SuperBlockOrder = {
   ],
   [SuperBlockStages.Extra]: [
     SuperBlocks.CodingInterviewPrep,
-    SuperBlocks.ProjectEuler
+    SuperBlocks.ProjectEuler,
+    SuperBlocks.RespWebDesign
   ],
-  [SuperBlockStages.Legacy]: [SuperBlocks.RespWebDesign],
+  [SuperBlockStages.Legacy]: [],
```

Comment:
This shouldn't be changed. The previous responsive web design is still considered legacy?

Reply:
I guess so. 
I arranged the superblocks by following [the design](https://github.com/freeCodeCamp/freeCodeCamp/issues/50412#issuecomment-1585086022) here